rfcs/2022-02-28-avatars.md

```diff
@@ -0,0 +1,14 @@
+# Include (Optional) Avatars in User Profiles
+
+1. Add SQL migration for database in `src/Simplex/Chat/Migrations`
+    - This will touch `contact_profiles` and `group_profiles`
+
+2. Add field to `User` in `Types.hs` allowing for null entry using `Maybe`
+
+3. Extend parsing in `Chat.hs` under `chatCommandP :: Parser ChatCommand`
+
+4. Update `UpdateProfile` in `Chat.hs` to accept possible display picture.
```

Comment:
we will need a separate command APIUpdateProfile that accepts JSON string:

/_profile {...}